### DISCUSSION TITLE
Hi excuse me~ this  library support MAUI?

### DISCUSSION THREAD

**Discussion by ghost** [SixLabors/ImageSharp#2484] [Q&A] [answered]
I add package in my MAUI project.
![image](https://github.com/SixLabors/ImageSharp/assets/15758574/3d8d8f03-2dfd-4937-9d86-4eb02d6ec661)

i am try use blur image. like this
![image](https://github.com/SixLabors/ImageSharp/assets/15758574/11cbdbdb-faaa-4eca-99ea-527f21f37171)

but this is no result... it is suck  `Image.Load`.
so how can i fix ?

**Comment by JimBobSquarePants** [comment]
ImageSharp.Drawing and Fonts betas are v2 compatible only.

**Reply by ghost** [reply]
so  it is available now?

**Reply by JimBobSquarePants** [reply, marked as answer]
You need to use the latest ImageSharp v2 not v3
